proto/storage/cluster.proto

```diff
@@ -280,22 +280,28 @@ message ClusterHealthStatus {
         HEALTHY       = 4;
     }
 
-    CollectorHealthInfo       collector_health_info             = 1;
+    CollectorHealthInfo        collector_health_info            = 1;
     AdmissionControlHealthInfo admission_control_health_info    = 8;
+    // scanner_health_info is filled when the scanner is deployed on a secured cluster (so called "local scanner").
+    // Please do not confuse this with the default scanner deployment on a central cluster.
+    ScannerHealthInfo          scanner_health_info        = 10;
```

Comment:
nit: I think the `= 10` needs to be further to the right.

Reply:
Fixed.